services/spar/src/Spar/Scim/User.hs

```diff
@@ -664,16 +725,18 @@ assertHandleNotUsedElsewhere uid hndl = do
 -- | Helper function that translates a given brig user into a 'Scim.StoredUser', with some
 -- effects like updating the 'ManagedBy' field in brig and storing creation and update time
 -- stamps.
-synthesizeStoredUser :: UserAccount -> ST.ValidExternalId -> Scim.ScimHandler Spar (Scim.StoredUser ST.SparTag)
-synthesizeStoredUser usr veid =
+synthesizeStoredUser :: UserAccount -> Scim.ScimHandler Spar (Scim.StoredUser ST.SparTag)
+synthesizeStoredUser usr =
   logScim
     ( logFunction "Spar.Scim.User.synthesizeStoredUser"
         . logUser (userId . accountUser $ usr)
         . maybe id logHandle (userHandle . accountUser $ usr)
         . maybe id logTeam (userTeam . accountUser $ usr)
-        . maybe id logEmail (veidEmail veid)
+        . maybe id logEmail (authIdEmail =<< userAuthId (accountUser usr))
```

Comment:
This is still the broken authId, right?  Move `authIdFromUser` above this line?

Reply:
Can't, because the authIdFromUser is wrapped by it. For logging it's fine to get the email from the authId from user I think

Reply:
I mean like this: 
e038c92